first_timers/first_timers.py

```diff
@@ -43,7 +48,28 @@ def get_fresh(old_issue_list, new_issue_list):
     old_urls = set(x['url'] for x in old_issue_list)
     return [x for x in new_issue_list if x['url'] not in old_urls]
 
+def get_primary_language(url):
+    """Returns the languages of the repo"""
+    github = Github(git_token)
+    match = re.match('https://github.com/(.*)/(.*)/issues/([0-9]*)', url)
+    if match is None:
+        raise RuntimeError('Format of API URLs has changed: ', api_url)
+    user, repo, issue_num = match.group(1), match.group(2), match.group(3)
+    language = g.get_user( user ).get_repo( repo ).language
```

Comment:
I'd rather not keep any white-space around the arguments, in keeping with PEP8. :)